<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Esto cambia el fondo de la ventana principal */
QMainWindow {
    background-color: #1e1e1e;
}

/* Esto cambia el color de todos los textos (Labels) */
QLabel {
    color: #ffffff;
}

/* Estilo para el SpinBox (cuadro de números) */
QSpinBox {
    background-color: #2d2d2d;
    color: white;
    border: 1px solid #3f3f3f;
    padding: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBoxPuntos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}
QProgressBar::chunk {
    background-color: #3498db;
    width: 20px;
}</string>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese la cantidad de productos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Faltante para el proximo descuento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Se ofrece un 1% de descuento cada 100 productos comprados con un tope de 50%.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRealizarCompra">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    border-radius: 5px;
    padding: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #27ae60;
}</string>
    </property>
    <property name="text">
     <string>Realizar compra</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCatalogo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Estilo de la caja principal */
QComboBox {
    background-color: #2d2d2d;
    color: white;
    border: 1px solid #3f3f3f;
    border-radius: 5px;
    padding: 5px 10px;
    min-width: 6em;
}

/* Estilo de la flechita */
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #3f3f3f;
    border-left-style: solid;
}

/* Estilo de la lista desplegable */
QComboBox QAbstractItemView {
    background-color: #1e1e1e;
    color: white;
    selection-background-color: #2ecc71; /* Color verde al pasar el mouse */
    selection-color: white;
    border: 1px solid #3f3f3f;
    outline: 0;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Ladrillo Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lamina</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chapa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cemento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arena (bolsa)</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listaDetalle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    background-color: #1a1a1a; /* Fondo casi negro */
    color: #2ecc71;            /* Texto verde esmeralda (estilo terminal) */
    border: 2px solid #3f3f3f; /* Borde gris oscuro */
    border-radius: 8px;        /* Bordes redondeados */
    padding: 10px;
    font-family: &quot;Consolas&quot;, &quot;Monospace&quot;, &quot;Courier New&quot;; /* Fuente de ancho fijo para que los números alineen bien */
    font-size: 11pt;
}

/* Estilo para cuando seleccionas un item de la lista */
QListWidget::item:selected {
    background-color: #2ecc71;
    color: black;
    border-radius: 4px;
}

/* Estilo para el scroll (barra de desplazamiento) */
QScrollBar:vertical {
    border: none;
    background: #2d2d2d;
    width: 10px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #3f3f3f;
    min-height: 20px;
    border-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Agregar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    border-radius: 5px;
    padding: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #27ae60;
}</string>
    </property>
    <property name="text">
     <string>Agregar al carrito</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSubtotal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtotal: $0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescuento">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descuento (0%): -$0.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TOTAL: $0.00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
